.github/workflows/main.yml

```diff
@@ -102,7 +102,7 @@ jobs:
         pip install -r requirements/test.txt fakeredis typing-extensions
         pip install hypothesis-python/[all]
     - name: Run tests
-      run: python -m pytest --numprocesses auto hypothesis-python/tests/
+      run: python -m pytest --numprocesses auto hypothesis-python/tests/ --ignore-glob=hypothesis-python/tests/quality/*.py
```

Comment:
The pytest documentation also mentions an `--ignore` flag, but I couldn't get it to do what I wanted, so I just resorted to globbing all of the test files in the quality directory.